profiles/repositories.py

```diff
@@ -102,16 +105,18 @@ def add(self, profile: Profile) -> Profile:
             self.db.session.rollback()
 
             message = exception.args[0]
+            log_msg = 'Unable to create profile with id {}. '.format(profile.id)
 
             if 'orcid' in message and profile.orcid:
-                LOGGER.info('Profile for ORCID %s appears to already exist', profile.orcid)
+                log_msg += 'Profile with ORCID {} already exists'.format(profile.orcid)
```

Comment:
abbreviations like `msg` are usually unnecessary, at least the style of this file (existing variables) suggests to use `message` instead

Reply:
The logger has an attribute called `msg` so I was following the same naming convention. I can change it if you prefer?

Reply:
good if it follows an established convention